src/SignupForm.js

```diff
@@ -4,6 +4,17 @@ import './SignupForm.css';
 
 const getAjaxUrl = url => url.replace('/post?', '/post-json?')
 
+function formatLinks(msg) {
+  let dom = document.createRange().createContextualFragment(msg);
```

Comment:
you might want to check for what kinds of exceptions this can throw. for instance what if `msg === undefined`?

Reply:
i updated to revert to the original `msg` string if any exceptions are thrown